.github/workflows/bump_version.yml

```diff
@@ -0,0 +1,43 @@
+name: Bump minor version
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  bump_version:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true && (github.event.number % 5 == 0)
```

Comment:
Honestly we can probably just bump the patch version every single PR instead of every 5. We are only a 100 or so PRs in and I imagine we'll only get 10-50 patch versions per minor version.